### DISCUSSION TITLE
The amount of unclosed processes is crazy

### DISCUSSION THREAD

**Discussion by ghost** [ultrafunkamsterdam/undetected-chromedriver#1533] [General]
I like this repo and have used it quite a bit, but my biggest issue is that it leaves so many instances of chrome open in memory. I haven't used this repo in a while but I just came back to it for a project I'm working on, and I was surprised to see the same issue still persists.

**Comment by ultrafunkamsterdam** [comment]
Without any code which can reproduce this issue, you could only wait for upcoming version which does not need a driver at all
